packages/runtime/test-runtime-utils/src/mocks.ts

```diff
@@ -500,6 +500,10 @@ export class MockFluidDataStoreRuntime extends EventEmitter
         return null;
     }
 
+    public async snapshotInternal(): Promise<ITreeEntry[]> {
```

Comment:
I'm not sure what's the point of empty implementation. If it get called, it will not fail, but result is still wrong, right?

Reply:
That was the implementation before I removed it. I believe it works because this is used in unit tests only and none of them call snapshot on this layer (or above).

Reply:
Makes sense - thanks!